lib/UI.js

```diff
@@ -102,6 +102,10 @@ define(function (require, exports, module) {
 
         $("#liveDevButton").click(function () {
             IframeBrowser.detachPreview();
+
+            // Hide attached live preview pane
+            $("#second-pane").css("width", "0");
+            $("#first-pane").css("width", "100%");
```

Comment:
I also wonder if you should actually do this using the `Splitter` api vs. just touching css values.